raiden/tests/utils/mediation_fees.py

```diff
@@ -118,11 +79,52 @@ def amount_without_fees(self) -> PaymentAmount:
         return PaymentAmount(self.total_amount - sum(self.mediation_fees))
 
 
-def get_initial_payment_for_final_target_amount(
-    final_amount: PaymentAmount, channels: List[NettingChannelState]
+def get_amount_with_fees(
+    amount_without_fees: PaymentWithFeeAmount,
+    balance_in: Balance,
+    balance_out: Balance,
+    schedule_in: FeeScheduleState,
+    schedule_out: FeeScheduleState,
+    receivable_amount: TokenAmount,
+) -> Optional[PaymentWithFeeAmount]:
+    """ Return the amount the transfer requires before fees are deducted.
+
+    This function is also used by the PFS. Therefore the parameters should not be Raiden state
+    objects.
+    """
+    assert (
```

Comment:
This is set for all FeeSchedules at raiden service start from what I can see. So this assertion will never hit for now.

In the future if we allow per channel `cap_fees` setting it should still not be possible to hit this as the cap fees setting from what I understand is per mediator, so should be the same for incoming and outgoing channel of each mediator?

Reply:
Yes, ideally the `cap_fees` would be a setting per mediator. It's hacky to put it into the `FeeScheduleState`, but we currently don't have a better place to put it that gets communicated to the PFS.

Reply:
Can you perhaps make a note of this in an issue? So we know this can be improved upon in the future? Naturally nothing to do with this PR.

Reply:
@karlb did you make an issue for this?

Reply:
https://github.com/raiden-network/raiden/issues/5120